snackbar/material/index.md

```diff
@@ -0,0 +1,18 @@
+# Aufgabenmaterial herunterladen
+
+Um das Aufgabenmeterial für [Kiosk](../) herunterzuladen, gib folgenden Befehl in _ein neues Terminal_ in deinem Codespace ein:
+
+```
+wget -O - https://inf-lab.dev/kiosk/material/public/scripts/setup.sh | bash
```

Comment:
This seems like the wrong url for multiple reasons:
1. It references `kiosk` in its pathname
2. The script will probably be stored in the `material/scripts/setup.sh` file